tests/app/submitter/test_converter.py

```diff
@@ -204,6 +204,25 @@ def test_convert_census_answers(self):
             self.assertEqual(answer_object['data'][3]['answer_instance'], 0)
             self.assertEqual(answer_object['data'][3]['value'], '63 Somewhere')
 
+    def test_convert_census_answers_multiple_answers(self):
+        with self.application.test_request_context():
+            routing_path = [location(group_id='personal details', block_id='about you')]
+            answers = [create_answer('name', ['Joe Bloggs', 'Fred Bloggs'], group_id='personal details', block_id='about you')]
```

Comment:
Not if you have a checkbox answer with a selection of names ;). I will make the answer look more checkboxy